requirements.txt

```diff
@@ -0,0 +1 @@
+git+git://github.com/david415/python-ftools.git#egg=ftools
```

Comment:
Should we add it/a fork to pypi? Is it simple enough to vendor? @mleinart mentioned an alternative approach - would that make more sense? I think some action to make this dependency be available in a less likely to blow up way is warranted here.